lib/ui/text.dart

```diff
@@ -106,184 +105,1306 @@ class FontWeight {
+  ///           FontFeature.subscript(),
+  ///         ],
+  ///       ),
+  ///     );
+  ///   }
+  /// }
+  /// ```
   ///
-  /// See also:
+  /// ![](https://flutter.github.io/assets-for-api-docs/assets/dart-ui/font_feature_subs.png)
+  /// {@end-tool}
   ///
-  ///  * <https://docs.microsoft.com/en-us/typography/opentype/spec/features_ko#onum>
-  const FontFeature.oldstyleFigures() : feature = 'onum', value = 1;
-
-  /// Use proportional (varying width) figures.
+  /// See also:
   ///
-  /// For fonts that have both proportional and tabular (monospace) figures,
-  /// this enables the proportional figures.
+  ///  * <https://docs.microsoft.com/en-us/typography/opentype/spec/features_pt#subs>
+  ///  * [FontFeature.scientificInferiors], which is similar but intended specifically for
+  ///    subscripts used in scientific contexts.
+  ///  * [FontFeature.superscripts], which is similar but for subscripting.
+  const FontFeature.subscripts() : feature = 'subs', value = 1;
+
+  // sups
+  /// Enable superscripts. (`sups`)
+  ///
+  /// This feature causes some fonts to change some glyphs to their
+  /// superscripted form. This may be more than just changing their
+  /// position. For example, digits might change to lining figures
+  /// (see [FontFeature.liningFigures]) in addition to being raised
+  /// and shrunk.
+  ///
+  /// It typically does not affect all glyphs, and so is not
+  /// appropriate for generally causing all text to be superscripted.
+  ///
+  /// This may override other features that substitute glyphs.
+  ///
+  /// {@tool sample --template=freeform}
+  ///
+  /// The Sorts Mill Goudy font supports the `sups` feature. It causes
+  /// digits to be smaller, superscripted, and changes them to lining
+  /// figures (so they are all the same height).
+  ///
+  /// ```dart
+  /// class Demo extends StatelessWidget {
+  ///   // The Sorts Mill Goudy font can be downloaded from Google Fonts (https://www.google.com/fonts).
+  ///   @override
+  ///   Widget build(BuildContext context) {
+  ///     return Text(
+  ///       'The isotope 238U decays to 206Pb',
+  ///       style: TextStyle(
+  ///         fontFamily: 'Sorts Mill Goudy',
+  ///         fontFeatures: <FontFeature>[
+  ///           FontFeature.superscript(),
+  ///         ],
+  ///       ),
+  ///     );
+  ///   }
+  /// }
+  /// ```
+  ///
+  /// ![](https://flutter.github.io/assets-for-api-docs/assets/dart-ui/font_feature_sups.png)
+  /// {@end-tool}
   ///
-  /// This is mutually exclusive with [FontFeature.tabularFigures].
+  /// See also:
   ///
-  /// The default behavior varies from font to font.
+  ///  * <https://docs.microsoft.com/en-us/typography/opentype/spec/features_pt#sups>
+  ///  * [FontFeature.subscripts], which is similar but for subscripting.
+  const FontFeature.superscripts() : feature = 'sups', value = 1;
+
+  // swsh
+  /// Enable swash glyphs. (`swsh`)
+  ///
+  /// Some fonts have beautiful flourishes on some characters. This
```

Comment:
Can we be a little more descriptive on this line? Beautiful flourishes sounds nice but a bit vague. The image really helps me here (and elsewhere too, they're great!), but maybe we could describe this as something like `This feature, when enabled, draws flourishes that continue the ending stroke of a letter.`? I'm not sure if it applies to beginning as well though.

Reply:
I've added more text explaining the concept, and a link to wikipedia.